docs/source/algorithms/LoadNexus-v1.rst

```diff
@@ -12,9 +12,25 @@ Description
 The algorithm LoadNexus will read the given Nexus file and try to
 identify its type so that it can be read into a workspace. The file name
 can be an absolute or relative path and should have the extension .nxs
-or .nx5. Currently only Nexus Muon Version 1 files are recognised, but
-this will be extended as other types are supported such as
-:ref:`algm-LoadNexusProcessed`.
+or .nx5.
+
+The type of Nexus file is identified as follows:
+
+* If the file has a group of class ``SDS`` of name ``"definition"`` or ``"analysis"`` 
```

Comment:
I found the reference to `analysis` hidden in the code for 

`Mantid::NeXus::getNexusEntryTypes(m_filename, entryName, definition)`.